src/components/sidebar/components/sidebar-title-heading/index.tsx

```diff
@@ -0,0 +1,16 @@
+import sSidebar from 'components/sidebar/sidebar.module.css'
+
+interface SidebarTitleHeadingProps {
```

Comment:
I'm probably behind on the convo on this, but in my previous interaction with the `MenuItem` interface, the title and heading were different. Is this just the compounding of those two as they share styles? Or whats the story with the `TitleHeading` combo?

Reply:
I put `Heading` in the names of `SidebarTitleHeading` and `SidebarSectionHeading` in hopes of implying that a heading HTML element is what gets rendered by these components. I think it also implies a relationship between the two since their names are similar, which I think is important because they're directly related to one another in both style (they currently have the same styles despite rendering different heading element levels) and hierarchy (`SidebarTitleHeading` is like a parent heading to `SidebarSectionHeading`) in the markup. I could add some comments in each file that help clarify their usage, would that be valuable?

Reply:
This seems like a hard naming problem 😅  I like the idea of capturing both the HTML `heading` element part of things, and the hierarchy here... 

... it seems like the design-side name of `Title` is maybe making this more complex? I think in the absence of that, we might reach for something like `SidebarHeading` & `SidebarSectionHeading`?

In any case, I'm on board with any of the names suggested here 👍

Reply:
Maybe this is a case where we need some terminology alignment. Where is `Title` being used on the design-side? Want to make sure I'm understanding where the confusion is stemming from. 😊

Reply:
+1 on getting alignment on this. Here's where I've seen it in design specs:

![CleanShot 2022-04-13 at 15 07 27@2x](https://user-images.githubusercontent.com/4624598/163252482-803b4856-4de4-4d69-b4cc-cb9fde7599cb.png)

What I find confusing is how the distinction between `title` and `heading` seems like it varies in dev contexts (`title` being the named HTML attribute, `heading` being HTML element) vs design contexts (I'm honestly not sure on exact definitions, I feel like `title` and `heading` are kind of interchangeable, but folks lean towards `title` sometimes to avoid confusion with `header` & `heading` sounding so similar).

Reply:
Ahh I see now! With title/heading _probably_ being interchangeable on the design side, and less so on the development side, I'd vote for the following names:

- Sidebar heading (code name: `SidebarHeading`)
- Section heading (code name: `SidebarSectionHeading`)

These code names are exactly what you proposed! 😊 What are your thoughts on waiting to rename until we've confirmed terminology alignment with design? I can go ahead and start that convo and then update these names in a follow-up PR.

Reply:
☝️ Sounds great! Very on board with keeping the names you have, & following up later once we have design alignment 💯

Reply:
Created an Asana ticket for discussing this with design! https://app.asana.com/0/1202110981600689/1202132404550212/f